README.md

```diff
@@ -6,15 +6,26 @@ We will be building an API for the purpose of accessing application data program
 
 Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
 
-## Kanban
+<!-- ## Kanban
 
 ### Link to your Trello Board here: https://trello.com/b/7yiHe1nI
 
 To keep track of the tasks involved in this project we're going to use a kanban board. Ensure that you work on one _ticket_ at time. You can click on the ticket to find out more information about what is required for the feature. A ticket is not considered complete unless both the happy path and errors response are handled. You can make use of the checklist on each ticket to keep track of the errors you want to handle. You can also make use of [error-handling.md](error-handling.md) to consider the error codes we may wish to respond with.
 
-**Please ensure you work through the tickets in numerical order.**
+**Please ensure you work through the tickets in numerical order.** -->
 
-## Git Branching and Pull Requests
+## Cloning
+
+Fork and clone this repo, and then:
+
+1.  run `npm install` in order to install all of the required dependencies.
+2.  Create two .env files in the root directory:
+
+    > **.env.development** Populate with '_PGDATABASE=<name_of_your_db>_'
```

Comment:
Could you let the person cloning know what the databases need to be called?
More chance of it going wrong otherwise